src/pytest_springfield/dottedname/foo/bar/bop.py

```diff
@@ -0,0 +1,14 @@
+from springfield import fields, Entity
+
+
+class Property(Entity):
+    name = fields.StringField()
+    nested = fields.EntityField(
+        'pytest_springfield.dottedname.foo.bar.bop.PropertyList'
```

Comment:
I also could not think of better alternatives :(

Improvements that I omitted on purpose, which maybe you want:

- require this dotted-name callable to actually be a real subclass of `Entity`, or
- require this dotted-name callable to implement `adapt()` (or some other thing like this)

Symbols should always be preferred, I just don't know how we could do meaningful symbols without huge hackery/monkeypatching.